### DISCUSSION TITLE
CapsLock Indicator - How do you save the settings ?

### DISCUSSION THREAD

**Discussion by gomo25mail** [jonaskohl/CapsLockIndicator#147] [General]
Hello;
CapsLock Indicator from [https://cli.jonaskohl.de/](url)
How do you save the settings ?

**Comment by jonaskohl** [comment]
Settings get auto-saved as you change them

**Reply by gomo25mail** [reply]
Thanks.
also
[https://github.com/jonaskohl/CapsLockIndicator/discussions/123](url)
